content/en/guides/integrations/diffusers.md

```diff
@@ -43,11 +43,11 @@ autolog(init=dict(project="diffusers_logging"))
         ```
 
 
-2. Use `autolog` to initialize a Weights & Biases run and automatically track the inputs and the outputs from [all supported pipeline calls](https://github.com/wandb/wandb/blob/main/wandb/integration/diffusers/autologger.py#L12-L72).
+2. Use `autolog` to initialize a W&B run and automatically track the inputs and the outputs from [all supported pipeline calls](https://github.com/wandb/wandb/blob/main/wandb/integration/diffusers/autologger.py#L12-L72).
```

Comment:
```suggestion
2. Use `autolog` to initialize a W&B Run and automatically track the inputs and the outputs from [all supported pipeline calls](https://github.com/wandb/wandb/blob/main/wandb/integration/diffusers/autologger.py#L12-L72).
```
At the first mention of a W&B-specific functionality like "W&B Run", the word after W&B should be capitalized. Afterward, you don't need to use W&B and the word should be lowercased like "run". Same thing for W&B Registry, W&B Sweeps, W&B Artifact, etc.